germlineVC.nf

```diff
@@ -402,12 +384,11 @@ process ConcatVCF {
     set variantCaller, idPatient, idSampleNormal, idSampleTumor, file("*.vcf.gz") into vcfConcatenated
     file("*.vcf.gz.tbi") into vcfConcatenatedTbi
 
-  when: ( 'haplotypecaller' in tools || 'mutect1' in tools || 'mutect2' in tools || 'freebayes' in tools ) && !params.onlyQC
+  when: 'haplotypecaller' in tools && !params.onlyQC
```

Comment:
I am a bit confused here, so it is germline calling, so all the somatic callers are removed, if I am correct.